content/post/190801-mockups.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Creating mockups for SONAR"
+date: 2018-08-01T17:41:58+02:00
+draft: false
+tags: ["mockups", "user interface", "user-centered design"]
+---
+
+The work on designing the user interface of SONAR is progressing. According to the adopted methodology (presented in the beginning of the [post about personas](/post/personas/)), we are proceeding in five layers: 1. Strategy, 2. Scope, 3. Structure, 4. Skeleton and 5. Surface.
```

Comment:
Should the layers be an ordered list?

Reply:
No, as it appears in the chapeau of the article.